### ISSUE TITLE
NullReferenceException with new v0.9.63 Achievements system

### ISSUE THREAD

**Issue by toxikman** [playgameservices/play-games-plugin-for-unity#2553]
This exception is happening in the anonymous callback within PlayGamesPlatform.ReportProgress, see mClient.LoadAchievements(ach => ...)
```
NullReferenceException: A null value was found where an object instance was required.
GooglePlayGames.PlayGamesPlatform+<ReportProgress>c__AnonStorey3.<>m__0 (GooglePlayGames.BasicApi.Achievement[] ach)
GooglePlayGames.OurUtils.PlayGamesHelperObject.Update ()
```
Happening frequently on all Android versions. We are using Unity 2017.4.25f1, GPGS 0.9.63

We don't know the exact line it's happening on because we can't reproduce it internally, but our crash logging system is reporting it from hundreds of end-users.

**Comment by olehkuznetsov** [comment]
Thanks for the bug reporting - this will be fixed in v0.9.64 soon.

By the way it is more optimal to use directly 
  PlayGamesPlatform.Instance.IncrementAchievement
  PlayGamesPlatform.Instance.UnlockAchievement
instead of ReportProgress.

As for ReportProgress first we need to find out which type of the achievement id is passed and to do this we need to load achievements first.

**Comment by toxikman** [comment]
Thanks for the tip!  We're now using SetStepsAtLeast and UnlockAchievement, since the game knows which type it is. Works great.

**Comment by sayiliromer** [comment]
On here, patch notes.
https://unity3d.com/unity/whats-new/2018.3.13
IL2CPP: Fixed default current culture not being detected correctly on macOS and iOS instead of the invariant culture. (1133961, 1135065)

I believe the bug that they fixed, changed the default behavior of all CultureInfo depended functions.  
Including double.Parse, float.Parse , System.Convert.ToDouble , string.ToUpper etc.

**Comment by olehkuznetsov** [comment]
Should be fixed in v0.9.64

**Comment by toxikman** [comment]
Thanks for the quick turn-around!
